lib/widget/loading_container.dart

```diff
@@ -167,3 +170,177 @@ class ListDetailShape extends StatelessWidget {
         ],
       );
 }
+
+const _shimmerGradient = LinearGradient(
```

Comment:
Move these shimmer-related stuff below to its own class.

Reply:
We need to document what the below classes do. Is it maintainable? If not I'd prefer to just use a library.

Reply:
Okay, I just built it from the official cookbook of Flutter - https://docs.flutter.dev/cookbook/effects/shimmer-loading.
That package is not working with the image.